activitypub/apmodels/webfinger.go

```diff
@@ -43,6 +45,16 @@ func MakeWebfingerResponse(account string, inbox string, host string) WebfingerR
 				Type: "text/html",
 				Href: accountIRI.String(),
 			},
+			{
+				Rel:  "avatar",
+				Type: logoType,
+				Href: logoIRI.String(),
+			},
+			{
+				Rel:  "media",
```

Comment:
According to https://microformats.org/wiki/rel-alternate#With_type it seems like `alternate` with the HLS content type might be the best match. There's no listing for `media`.

Reply:
Yeah, even I couldn't find one. Sure, will make the change, and based on the type _application/x-mpegURL_. It will associate with the context which is the stream link.